src/assets/styles/app.styl

```diff
@@ -156,8 +156,8 @@ a:hover .idg-logo-idrc {
         margin: 2rem 0;
         border-radius: 0.2rem;
 
-        figure {
-            margin: 0;
+        div:first-child {
+            padding-block: : 1rem;
```

Comment:
This syntax issue appears to be breaking the build.

Reply:
I've addressed this syntax error.